Code/PgSQL/rdkit/mol_op.c

```diff
@@ -376,7 +378,7 @@ fmcs_mol2s_transition(PG_FUNCTION_ARGS) {
     }
     if(PG_ARGISNULL(0) && !PG_ARGISNULL(1)) { // first call
 ///elog(WARNING, "fmcs_mol2s_transition() called first time");
-        CROMol mol = PG_GETARG_DATUM(1);
+        CROMol mol = (CROMol)PG_GETARG_DATUM(1);
```

Comment:
Crap, I'm sorry, I missed that.
I thought this was still in adapter.cpp.
That's what I get for reviewing code on my iPad.

It's fine as is.

-greg


On Thursday, 4 February 2016, Paolo Tosco <notifications@github.com> wrote:

> In Code/PgSQL/rdkit/mol_op.c
> <https://github.com/rdkit/rdkit/pull/752#discussion_r51847379>:
>
> > @@ -376,7 +378,7 @@ fmcs_mol2s_transition(PG_FUNCTION_ARGS) {
> >      }
> >      if(PG_ARGISNULL(0) && !PG_ARGISNULL(1)) { // first call
> >  ///elog(WARNING, "fmcs_mol2s_transition() called first time");
> > -        CROMol mol = PG_GETARG_DATUM(1);
> > +        CROMol mol = (CROMol)PG_GETARG_DATUM(1);
>
> Will do. I used the C-style cast because the file extension was .c rather
> than .cpp.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rdkit/rdkit/pull/752/files#r51847379>.
>